### DISCUSSION TITLE
Consider including data license info via HTTP response headers

### DISCUSSION THREAD

**Discussion by imathews** [redivis/meta#253] [Feature requests]
When reading data from dataset(s), it might be good to show the user info about license information for the associated datasets. This could be returned via HTTP response headers; e.g., `X-Redivis-Info` (more generic, could send other info in the future) or `X-Redivis-License`. And then the python / R client libraries could print this to the console (with perhaps the ability to disable such logging).

It's worth considering the degree to which this would create excess noise for users, vs. being helpful. It may also make sense to not print within Redivis notebooks, since in this scenario the data aren't leaving Redivis, and citation information could be presented more front-and-center in the user interface.

**Comment by ben-domingue** [comment]
I think if we could include a brief desciption of the license (e.g., CC BY NC as per the below) that would be useful. 
![image](https://github.com/user-attachments/assets/89219247-023e-4310-8c40-c0d870b18042)

**Reply by ben-domingue** [reply]
this obviously would only apply to the CC licenses, but a brief <10 word 'name' is available for all of them
